plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -177,14 +191,24 @@ type VolumeFilter struct {
 	FilterPersistentVolume func(pv *v1.PersistentVolume) (id string, relevant bool)
 }
 
+func NewMaxEBSVolumeCountPredicate(pvInfo PersistentVolumeInfo, pvcInfo PersistentVolumeClaimInfo) algorithm.FitPredicate {
```

Comment:
I still prefer the old `NewMaxPDVolumeCountPredicate`.

Actually there is no need to add three new-added methods, which have the same exact apis.

Reply:
Similar to what @dixudx said, I wonder why we need to wrap `newMaxPDVolumeCountPredicate`?

Reply:
@dixudx  @bsalamat  Thank you for your suggestion。 I add a new param to incorporate these three cases. please view again.